server/settings.py

```diff
@@ -8,59 +8,38 @@
 #     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
 import os
 
+from scrapy.utils.project import get_project_settings
```

Comment:
`scrapy.settings` was integrated into `server.settings`. Reference to google api token was removed for security. It is now in a `.env` file that's source controlled. A template file for `.env` was added in https://github.com/comp-strat/scraping_server/pull/29.